src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void getDeadlineForDisplayTest(){
```

Comment:
Does not follow standard test naming convention. Correct me if I'm wrong if this isn't the standard convention

`featureName_situation_expectedResult`. 

E.g. `getDeadline_validFormat_returnsFormattedDate`

Reply:
Oh yea you are right, I'm so frustrated while setting up the Gradle and making the JUnit tests, until a point that I only want it to work, which makes me forget the naming convention haha.